src/ext/java/org/opentripplanner/ext/stopconsolidation/StopConsolidationModule.java

```diff
@@ -74,10 +75,10 @@ private TripPattern modifyStopsInPattern(
   public static StopConsolidationModule of(
     TransitModel transitModel,
     StopConsolidationRepository repo,
-    File file
+    DataSource ds
   ) {
-    LOG.info("Reading stop consolidation information from '{}'", file.getAbsolutePath());
-    var groups = StopConsolidationParser.parseGroups(file);
+    LOG.info("Reading stop consolidation information from '{}'", ds);
+    var groups = StopConsolidationParser.parseGroups(ds.asInputStream());
```

Comment:
`ds.asInputStream()` returns a stream that needs to be closed.

Reply:
Adressed in f671f54e92444fa8533e5669f36ecd13ef00204e.